packages/features/bookings/lib/create-booking.ts

```diff
@@ -3,6 +3,12 @@ import { post } from "@calcom/lib/fetch-wrapper";
 import type { BookingCreateBody, BookingResponse } from "../types";
 
 export const createBooking = async (data: BookingCreateBody) => {
-  const response = await post<BookingCreateBody, BookingResponse>("/api/book/event", data);
+  const response = await post<
+    Omit<BookingCreateBody, "startTime" | "endTime">,
+    BookingResponse & {
+      startTime: string;
+      endTime: string;
```

Comment:
The response of a regular fetch request can't have Date as value for any prop, so we accordingly adjust the types.